util/Endpoint.php

```diff
@@ -282,7 +282,7 @@ private function extractUrl(array $paths): string
                     );
                     $this->addNamespace('OpenSearch\Common\Exceptions\RuntimeException');
                 } else {
-                    $params .= sprintf("%s\$%s = \$this->%s ?? null;\n", $tab8, $part, $part);
+                    $params .= sprintf("%s\$%s = \$this->%s ?? null;", $tab8, $part, $part);
```

Comment:
- A minor fix is done for the generator here.

- Apart from that, **all other changes are generated** using util/GenerateEndpoints.php. Only endpoints with generated code equivalent to the existing code are included in this PR. Endpoints that are not accurately generated will be addressed in follow-up PRs.